config_get.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/weaveworks/footloose/pkg/config"
+
+	"github.com/spf13/cobra"
+)
+
+var getConfigCmd = &cobra.Command{
+	Use:   "get",
+	Short: "Get config file information",
+	RunE:  getConfig,
+}
+
+var getOptions struct {
+	config string
+}
+
+func init() {
+	getConfigCmd.Flags().StringVarP(&getOptions.config, "config", "c", Footloose, "Cluster configuration file")
+	configCmd.AddCommand(getConfigCmd)
+}
+
+func getConfig(cmd *cobra.Command, args []string) error {
+	c, err := config.NewConfigFromFile(getOptions.config)
+	if err != nil {
+		return err
+	}
+	var detail interface{}
+	detail = c
+	if len(args) > 0 {
+		detail, err = config.GetValueFromConfig(args[0], c)
+		if err != nil {
+			log.Println(err)
+			return fmt.Errorf("Failed to get config detail")
+		}
+	}
+	res, err := json.MarshalIndent(detail, "", "  ")
+	if err != nil {
+		log.Println(err)
+		return fmt.Errorf("Cannot convert result to json")
+	}
+	fmt.Printf("%s", res)
```

Comment:
I haven't tested it but I think this will output strings with a `""`, ie `config get name` may output

```
"cluster"
```
but it should be
```
cluster
```

Reply:
You are totally right